lib/dradis/plugins/nexpose/list_formatter.rb

```diff
@@ -0,0 +1,42 @@
+module Dradis::Plugins::Nexpose
+  class ListFormatter
+    def format_list(source)
+      # Add <root> node in case the source is an invalid xml
+      xml = Nokogiri::XML("<root>#{source}</root>") { |conf| conf.noblanks }
+      xml = xml.at_xpath('./root/ContainerBlockElement')
+
+      return source unless xml
+
+      if xml.xpath('./UnorderedList | ./OrderedList').any?
+        format_nexpose_list(xml)
+      else
+        source
+      end
+    end
+
+    private
+
+    def format_nexpose_list(xml, depth = 1)
+      xml.xpath('./UnorderedList | ./OrderedList').map do |list|
+        list_type = list.name == 'UnorderedList' ? '*' : '#'
```

Comment:
```suggestion
        list_item_type = list.name == 'UnorderedList' ? '*' : '#'
```
Or `list_item_element` / `list_element` ? Something a bit more descriptive